assert/assertions.go

```diff
@@ -1501,6 +1501,50 @@ func InEpsilonSlice(t TestingT, expected, actual interface{}, epsilon float64, m
 	return true
 }
 
+// InEpsilonMapValues is the same as InEpsilon, but it compares all values between two maps. Both maps must have exactly the same keys.
+func InEpsilonMapValues(t TestingT, expected, actual interface{}, epsilon float64, msgAndArgs ...interface{}) bool {
+	if h, ok := t.(tHelper); ok {
+		h.Helper()
+	}
+	if expected == nil || actual == nil ||
+		reflect.TypeOf(actual).Kind() != reflect.Map ||
+		reflect.TypeOf(expected).Kind() != reflect.Map {
```

Comment:
Remove the check of the kind of _actual_.

Instead, just check the kind of _expected_ and check that actual has the same type with call `IsType(t, expected, actual, msgAndArgs...)`.

Reply:
Thank you for review, kindly please check again